packages/bahmni-widgets/src/search/commonSearch/styles/CommonSearchWidget.module.scss

```
@@ -92,3 +94,40 @@
+
+.searchSummary {
+  display: flex;
+  align-items: center;
+  padding-bottom: $spacing-04;
+  padding-top: 0;
+  gap: $spacing-05;
+}
+
+.summaryLeft {
+  @include type-style("body-compact-01");
+  flex: 0 0 70%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)